src/common/datomish/pair_chan.cljc

```diff
@@ -71,6 +71,13 @@
      (consume-pair (cljs.core.async/<! ~pc-chan))
      (consume-pair (clojure.core.async/<! ~pc-chan))))
 
+(defmacro <??
+  "Takes from the channel if it's non-nil."
+  [pc-chan]
+  `(let [c# ~pc-chan]
+     (when c#
+       (datomish.pair-chan/<? c#))))
```

Comment:
I think I got it, thanks! This is neat stuff.

If `c#` will be bound to a channel once expanded, shouldn't it be called something like `<c#`? Or is that convention only for functions that return channels? In which case, shouldn't it be `[<pc-chan]`?